pubsub/google/cloud/pubsub/topic.py

```diff
@@ -435,12 +462,27 @@ def publish(self, message, **attrs):
 
         :type attrs: dict (string -> string)
         :param attrs: key-value pairs to send as message attributes
+
+        :rtype: None
```

Comment:
1. `None` is not a type. (`NoneType` is the type)
1. If it doesn't return anything you don't need a returns/rtype block